src/viur/shop/modules/vat.py

```diff
@@ -16,3 +16,8 @@ def adminInfo(self) -> dict:
         admin_info = super().adminInfo()
         admin_info["icon"] = "cash-stack"
         return admin_info
+
+    def get_vat_by_value(self, vat_percent: float) -> db.Key:
+        if not (skel := self.viewSkel().all().mergeExternalFilter({"rate": vat_percent}).getSkel()):
+            raise errors.NotFound("Vat not found")
+        return skel
```

Comment:
The naming should be consistent:
```suggestion
    def get_vat_skel_by_rate(self, rate: float) -> db.Key:
        if not (skel := self.viewSkel().all().mergeExternalFilter({"rate": rate}).getSkel()):
            raise errors.NotFound("VatSkel not found")
        return skel
```


BUT ...